src/test/scala/com/campudus/tableaux/api/content/DeleteTest.scala

```diff
@@ -8,7 +8,7 @@ import org.junit.runner.RunWith
 import org.vertx.scala.core.json.Json
 
 @RunWith(classOf[VertxUnitRunner])
-class DeleteTest extends TableauxTestBase {
+class DeleteRowTest extends TableauxTestBase {
```

Comment:
Would be good to rename the file into `deleteTests.scala` or similar (lowercase letter is for "multiple classes / objects in here")